features/json-raw.yml

```
@@ -0,0 +1,8 @@
+name: JSON source text access
+description: With JSON source text access, you can use a `reviver` callback to post-process data in `JSON.parse()`, and you can pre-process `JSON.rawJSON` using `replacer` in `JSON.stringify()`. This is useful for lossless number serialization for BigInt numbers, for example.
```

This is tough to explain! I've made an attempt to try to explain what "source text access" refers to and came up with this version:

```suggestion
description: The JSON source text access API is the `JSON.rawJSON` static method, `JSON.isRawJSON()` static method, a source context parameter for the `JSON.parse()` reviver callback, and `rawJSON` value handling in `JSON.stringify()`. You can use this API for lossless serialization.
```

I'm not sure it's any better. Take it or leave it. 😄